scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -116,6 +116,11 @@ abstract class H2OMOJOModel
 
   def getFeatureImportances(): DataFrame = $(featureImportances)
 
+  def getHexMojoModel(): _root_.hex.genmodel.MojoModel = {
```

Comment:
maybe avoid using "hex"? we could call it "unwrapMojoModel" instead - since H2OMojoModel is a wrapper of MOJO

Reply:
Updated, thank you!